tools/cbmc/proofs/Makefile.template

```diff
@@ -110,11 +113,13 @@ $(ENTRY).goto: $(ENTRY)5.goto
 # Rules for running CBMC
 
 goto:
```

Comment:
Do you not want to do the `_goto`, `_report`, etc trick that we do for other repositories, so that people can just run `make report` in the proof directory and have it 'just work' with litani?

Reply:
No, we are not changing the targets themselves.  We are using litani to run `make goto`, we are not using `make goto` to run litani as we do in the starter kit.